wallet/walletrpc.c

```diff
@@ -197,7 +197,7 @@ static void json_withdraw(struct command *cmd,
 	struct bitcoin_tx *tx;
 	bool withdraw_all = false;
 
-	if (!json_get_params(buffer, params,
+	if (!json_get_params(cmd, buffer, params,
 			     "destination", &desttok,
 			     "satoshi", &sattok,
 			     NULL)) {
```

Comment:
Oops, the `command_fail` on the next line should probably be removed also.

Reply:
Oh, yeah, my API change in another PR.

Reply:
Rebased...